Makefile

```diff
@@ -53,12 +53,28 @@ GO_SUBDIRS += cmd internal apis
 # ====================================================================================
 # Setup Kubernetes tools
 
-KIND_VERSION = v0.15.0
-UP_VERSION = v0.16.1
+KIND_VERSION = v0.21.0
+UP_VERSION = v0.28.0
 UP_CHANNEL = stable
-UPTEST_VERSION = v0.7.0
+UPTEST_VERSION = v0.11.1
+UPTEST_LOCAL_VERSION = v0.12.0-2.g635fa3e
+UPTEST_LOCAL_CHANNEL = main
+KUSTOMIZE_VERSION = v5.3.0
+YQ_VERSION = v4.40.5
+UXP_VERSION = 1.14.6-up.1
+
 -include build/makelib/k8s_tools.mk
 
+# uptest download and install
+UPTEST_LOCAL := $(TOOLS_HOST_DIR)/uptest-$(UPTEST_LOCAL_VERSION)
+
+$(UPTEST_LOCAL):
+	@$(INFO) installing uptest $(UPTEST_LOCAL)
+	@mkdir -p $(TOOLS_HOST_DIR)
+	@curl -fsSLo $(UPTEST_LOCAL) https://s3.us-west-2.amazonaws.com/crossplane.uptest.releases/$(UPTEST_LOCAL_CHANNEL)/$(UPTEST_LOCAL_VERSION)/bin/$(SAFEHOST_PLATFORM)/uptest || $(FAIL)
+	@chmod +x $(UPTEST_LOCAL)
+	@$(OK) installing uptest $(UPTEST_LOCAL)
+
```

Comment:
This is a temporary workaround till https://github.com/upbound/build/pull/255 is merged.